src/scenes/TasksView/components/TaskCard/index.tsx

```diff
@@ -32,9 +35,12 @@ function TaskCard({task, completed}: TaskCardProps) {
             </Badge>
           )}
         </div>
+       
       </Card>
     </div>
   );
 }
 
 export default TaskCard;
+
```

Comment:
Some unnecessary changes, probably by mistake you've edited the file, lets undo these changes

Reply:
This entire file*

Reply:
This is now fixed